plugins/woocommerce/src/Blocks/BlockTypes/AccordionHeader.php

```diff
@@ -0,0 +1,34 @@
+<?php
+namespace Automattic\WooCommerce\Blocks\BlockTypes;
+
+/**
+ * AccordionHeader class.
+ */
+class AccordionHeader extends AbstractBlock {
+	/**
+	 * Block name.
+	 *
+	 * @var string
+	 */
+	protected $block_name = 'accordion-header';
+
+    /**
+	 * Get the frontend style handle for this block type.
+	 *
+	 * @return string[]|null
+	 */
+	protected function get_block_type_style() {
+		return [ 'wc-blocks-style' ];
```

Comment:
Is it necessary? I think that the CSS that we added doesn't depend on `wc-blocks-style`. For this reason, I think that we can return `null`.

Reply:
Done as part of https://github.com/woocommerce/woocommerce/pull/54684/commits/8bd13083d60cac5ba12e07cae4e7d4a5c70691d0